client/i18n/locales/english/translations.json

```diff
@@ -16,7 +16,7 @@
     "claim-cert": "Claim Certification",
     "save-progress": "Save Progress",
     "accepted-honesty": "You have accepted our Academic Honesty Policy.",
-    "agree": "Agree",
+    "agree-honesty": "I accept freeCodeCamp's Academic Honesty Policy",
```

Comment:
This needs to be grammatically correct. Users are not "accepting" but "agreeing". If you change the word to agree, I think the sentence needs to become very verbose, which could be tricky in other languages. 

The verb/action could be "I agree" and just enough.

Reply:
Should I include `freeCodeCamp's Academic Honesty Policy` as screen only text?

Originally, Bruce recommended "freeCodeCamp's Academic Honesty Policy" to make it more accessible.

---

Side-note: should I change `"accepted-honesty": "You have accepted our Academic Honesty Policy.",` to `"accepted-honesty": "You have agreed to our Academic Honesty Policy.",`. or I am confusing the intent?

Reply:
Yeah, I think you should change it back. As for the the screen-only text it should be fine to use your suggestion.

Reply:
Ya, using "agree" instead of "accept" probably makes more sense. Regardless, we should use the same verb for both the initial button text and the button text after they click it.

Personally, I think the initial button text should be verbose for everyone and should have something like "I agree to freeCodeCamp.org's Academic Honesty Policy" visible to everyone. But if you only want "I agree" to be visible then you can add the rest as sr-only text.

Reply:
Let me bring this to i18n team, hopefully there will be no problem with having it for everyone. 👍

Reply:
Sorry for prolonging this, and thanks for your patience, Mrugush, Bruce, and Yoko. Updated it. 
![image](https://user-images.githubusercontent.com/88248797/214867869-9083af78-4f6f-46d9-9d7a-de345dc030dc.png)

Reply:
Hi @Sboonny This is a minor thing, but while you're updating this, could you change the email to "support@freecodecamp.org"? Campers will get a much faster response if they email that address instead.

Reply:
Done 👍.